qa/tasks/cephfs/test_fstop.py

```diff
@@ -112,3 +114,22 @@ def verify_fstop_metrics(metrics):
         self.mount_b.mount_wait(cephfs_name=self.fs.name)
 
         self.assertTrue(valid)
+
+    def test_version(self):
```

Comment:
Do we need this test since we already vet the python version during cmake?

Reply:
[1] https://bugzilla.redhat.com/show_bug.cgi?id=2083468#c27
I'm confused, as [1] happened at runtime. Would the cmake changes prevent [1] from happening ?

Reply:
We want the cmake fix to error out when the python version is not supported.

Reply:
ok fine with me. So I'll drop the `test_version` function. Do you want to keep the `--version` (dump_version) code?

Reply:
Not required. I would add a note in the docs too and maybe inform ceph-users list that this constraint is upcoming,